src/convert.js

```diff
@@ -128,72 +129,79 @@ function convertCamelCaseToKebabCase(string) {
 }
 
 function exportToJS(
-  resolvedTokensByCategory,
+  tokensByCategory,
```

Comment:
This is fine. When many function attributes I prefer object of properties or curried functions.